package.json

```diff
@@ -55,7 +58,12 @@
     "mkdirp": "^0.5.1",
     "png-crop": "0.0.1",
     "pngjs": "^2.3.1",
+    "react": "15.4.1",
```

Comment:
Should we remove this from devDepedencies at the same time? I suppose it won't matter once we actually do the plugin stuff we talked about.